### PR TITLE
Prac 8 landing page frontend

### PR DESCRIPTION
**Pull request by mynewyear**
## Requirements
Desktop and responsive layout for Landing page
Two buttons: Parent and Tutor.
Clicking on any of the buttons opens the Login/Register page.

## Notes

_Any additional info_

## Test Steps

_Instructions on how to test the changes_

## UI/UX Outcome

_Attach a screenshot here_
Screen Shot 2024-03-24 at 11 58 53 PM
Screen Shot 2024-03-25 at 12 09 03 AM

Screen Shot 2024-03-26 at 8 36 21 PM

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -21,6 +22,15 @@ const App = () => {
             <Routes>
                 <Route path="/" element={<Layout />}>
                     {/* <Route index element={<HomePage />} /> */}
+                    <Route
+                        index
+                        element={
+                            <LandingPage
+                                parentButtonText="I'm a parent"
```
Comment (IrynaKolh) [inline src/App.tsx]: Don't neet to pass this props, it is static data, it doesn't any effect.

File: src/pages/LandingPage.tsx [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+                    justifyContent="center"
+                >
+                    <Button
+                        size="landing"
+                        variant="buttonTeal"
+                        width={{ base: '90%', sm: '80%', lg: '350px' }}
+                        fontWeight="bold"
+                        onClick={toRegistration}
+                    >
+                        {parentButtonText}
```
Comment (IrynaKolh) [inline src/pages/LandingPage.tsx]: Here you can use "I'm a parent" or "I'm a tutor" as text

**Review by IrynaKolh — COMMENTED** [review]
Main background must be full page size.

File: src/App.tsx (hunk 1 above)
Comment (mynewyear) [reply src/App.tsx]: done

File: src/pages/LandingPage.tsx (hunk 2 above)
Comment (mynewyear) [reply src/pages/LandingPage.tsx]: done

**Review by IrynaKolh — APPROVED** [review]
LGFM

**Review by oxangyal — APPROVED** [review]
LGFM

**Pull request merged**